prisma/schema.prisma

```diff
@@ -2,10 +2,55 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-  provider = "prisma-client-js"
+    provider = "prisma-client-js"
 }
 
 datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
+    provider = "postgresql"
+    url      = env("DATABASE_URL")
```

Comment:
I recommend installing [Prisma's VS Code extension](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma) to autoformat to its preferred spacing.